api/bazel/repositories.bzl

```diff
@@ -287,3 +288,82 @@ go_proto_library(
 )
         """,
     )
+
+    _com_github_grpc_grpc()
+
+# TODO(bplotnick): port repository_locations.bzl repository management logic to here
+def _com_github_grpc_grpc():
+    git_repository(
+        name = "com_github_grpc_grpc",
+        remote = "https://github.com/grpc/grpc.git",
+        commit = GRPC_SHA,
+    )
+
+    # Rebind some stuff to match what the gRPC Bazel is expecting.
+    native.bind(
+        name = "protobuf_headers",
+        actual = "@com_google_protobuf//:protobuf_headers",
+    )
+
+    native.bind(
+        name = "libssl",
+        actual = "@boringssl//:ssl",
+    )
+
+    # Envoy requires this bind
+    native.bind(
+        name = "ssl",
+        actual = "@boringssl//:ssl",
+    )
+
+    if "boringssl" not in native.existing_rules():
+        native.http_archive(
+            name = "boringssl",
+            # chromium-68.0.3440.75
+            url = "https://boringssl.googlesource.com/boringssl/+archive/372daf7042ffe3da1335743e7c93d78f1399aba7.tar.gz",
+        )
+
+    native.bind(
+        name = "ares",
+        actual = "@com_github_cares_cares//:ares",
+    )
+
+    native.bind(
+        name = "cares",
+        actual = "@com_github_cares_cares//:ares",
+    )
+
+    if "com_github_cares_cares" not in native.existing_rules():
```

Comment:
What do you think about the @jmillikin-stripe ask regarding setting up the `repository_locations.bzl` infrastructure in `api/bazel` as well as in `bazel`?